src/chain/chain.rs

```diff
@@ -691,6 +691,7 @@ impl Chain {
     }
 
     /// Returns a set of elders we should be connected to.
+    // WIP: should we remove potential duplicates?
```

Comment:
We should also not include `new_info.member_nodes` here, but that can maybe be done in other PR.

Reply:
Yep I agree. Ok if I add that change to the Part 3 PR so as to not invalidate the soak testing?

Reply:
Sure. It might event be done as a completely separate PR. Up to you.

Reply:
Thanks. In that case I'll raise a separate PR I think :)

Reply:
Removed in https://github.com/maidsafe/routing/pull/1898